resources/css/education.css

```diff
@@ -0,0 +1,182 @@
+@font-face {
+  font-family: OpenSans-SemiBold;
+  src: url(fonts/OpenSans-SemiBold.ttf)
+}
```

Comment:
This goes for the every page that we are working on: we are going to keep a style of 4 spaces for each tab (no tab charcaters or 2 spaces), even in CSS and JS files. All of the 2-spaces for tabs in this file should be changed to be 4 spaces